db/migrate/20241205142318_add_more_native_columns_to_contacts.rb

```diff
@@ -0,0 +1,21 @@
+class AddMoreNativeColumnsToContacts < ActiveRecord::Migration[8.0]
+  def change
+    change_table :contacts do |t|
+      t.string :status, null: false, default: 'active'
+      t.string :language_code
+      t.string :gender
+      t.date :date_of_birth
+
+      t.string :iso_country_code
```

Comment:
iso_country_code should be not null

Reply:
I did that, but reverted it because if it doesn't allow because we need to have data migration before we can mark it as not null.